docs/src/main/sphinx/sql/drop-function.md

```diff
@@ -0,0 +1,42 @@
+# DROP FUNCTION
+
+## Synopsis
+
+```text
+DROP FUNCTION [ IF EXISTS ] routine_name
```

Comment:
This needs to include the parameter types
```
DROP FUNCTION [ IF EXISTS ] routine_name ( [ [ parameter_name ] data_type [, ...] ] )

Reply:
Updated